src/lib/y2partitioner/widgets/overview.rb

```diff
@@ -131,21 +139,18 @@ def unused_items
         item_for(:unused, _("Unused Devices"), icon: Icons::UNUSED)
       end
 
-      def item_for(id, title, icon: nil, subtree: [])
-        args = [Id(id)]
-        args << term(:icon, icon) if icon
-        args << title
-        args << open?(id)
-        args << subtree
-        Item(*args)
+      def item_for(id, label, widget: nil, icon: nil, subtree: [])
+        log.info "id #{id} label #{label}"
```

Comment:
Later I'd decrease it `debug` only, if the system has 2000 disks (I had a real bug report with that amount) the log would be huge just after few clicks in the UI.